### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by slintes**
Fix paths after merging cluster-up repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Getting Started with a multi-node Kubernetes Provider
 
-Download cluster-up repo
+Download this repo
 ```
-git clone https://github.com/davidvossel/cluster-up.git
-cd cluster-up
+git clone https://github.com/kubevirt/kubevirtci.git
```
Comment (davidvossel) [inline README.md]: oops. ha, good catch.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+cd kubevirtci/cluster-up
```
Comment (davidvossel) [inline README.md]: I think this should just be ```cd kubevirtci```

All the scripts are meant to be acted on in the top level kubevirtci dir (where the make target lives), not within cluster-up directly. The scripts do use absolute paths though, so it's likely that they'll work fine regardless of what directory they are executed in. 

However ```make clusterup``` only works in kubevirtci, not kubevirtci/cluster-up

**Comment by slintes** [comment]
@davidvossel yep, saw it. Will revisit this and your related PRs as soon as I'm done with another task

**Comment by slintes** [comment]
updated, sorry for delay and confusion about into what directory to cd, somehow I saw the Makefile in cluster-up

**Pull request merged**